scss/datetime/_layout.scss

```diff
@@ -118,4 +139,95 @@
 
     }
 
+    // Infinite timepicker
+    // Header
+    .k-time-header { // Same as the calendar header
```

Comment:
Does the same reasoning apply to the calendar component? Using a `<button>` might be better for the calendar accessibility, too.